src/main/java/com/squareup/protoparser/EnumConstantElement.java

```diff
@@ -47,4 +45,38 @@ public static EnumConstantElement create(String name, int tag, String documentat
     }
     return builder.append(";\n").toString();
   }
+
+  public static final class Builder {
+    private String name;
+    private Integer tag;
+    private String documentation = "";
+    private final List<OptionElement> options = new ArrayList<>();
+
+    private Builder() {
+    }
+
+    public Builder setName(String name) {
```

Comment:
Need the `set` prefix? Effective Java 2e doesn't. I slightly don't like it, but recognize that I'm odd-man out